README.md

```diff
@@ -33,10 +33,25 @@ Properties
 - *pathToJsonFile* (OPTIONAL) - project relative path to your json file, default is *deploy/marathon.json*
 - *verificationTimeout* (OPTIONAL) - timeout when querying Marathon to verify that there are no pending deployments left, default is 90 seconds
 - *deploymentRequestTimeout* (OPTIONAL) - timeout for the initial deployment request, default is 5 seconds
+- *jvmOverhead* (OPTIONAL) - number of MBs that the container needs on top of JVM app memory (default is 200)
```

Comment:
Add `jvmMem` description here. Also write here that `mem` and `jvmMem` are exclusive.